Gemfile

```diff
@@ -5,8 +5,10 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in loop_client-client.gemspec
 gemspec
 
+gem 'pry'
```

Comment:
Please use `group :test, :development do ... end`

Reply:
in gem's we have not groups